examples/limit-requests/Readme.md

```diff
@@ -0,0 +1,32 @@
+### Example to limit requests
+
+This is a working example to limit the concurrency of requests.
+
+#### Motivation
+
+Using eth-fun without any concurrency can choke the RPC endpoint.
+
+#### Overview
+
+This example uses the [p-limit library](https://github.com/sindresorhus/p-limit) to limit concurrency.
+
+```js
+const pool = pLimit(concurrency);
```

Comment:
`concurrency` isn't defined.

Reply:
maybe we should run this example with test-readme-md

Reply:
I didn't mean for this snippet to be a fully working example. The full code is in index.js. If this is confusing we can remove it.

I did a quick test and we are not able to import libraries in test-readme-md. So, making the snippet complete won't help.

Reply:
ok fair enough